app/models/user.rb

```diff
@@ -419,6 +419,7 @@ def seed_aspects
     self.aspects.create(:name => I18n.t('aspects.seed.friends'))
     self.aspects.create(:name => I18n.t('aspects.seed.work'))
     aq = self.aspects.create(:name => I18n.t('aspects.seed.acquaintances'))
+    aq.chat_enabled = true # enable chat aspect in acquaintances
```

Comment:
Why? You didn't mention this in the description of the PR and also the commit message doesn't mention this. (Looks like a more or less unrelated change)